src/abis/permit2.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
can't this be exported by the permit2 sdk?

Reply:
it can, but we'd need to modify our build process to generate types for it. I think it's simpler to add a pared down version here, in `src/abis`, which are already being pre-processed to generate types. See the `contracts:compile:abi` script.